### PR TITLE
:lipstick: (UI): Made the maps and ticket list displays uniform across applications

### PR DESCRIPTION
**Pull request by LeonardoWessels**

### CODE REVIEW

File: libs/admin-app/dashboard/src/lib/dashboard-overview/dashboard-overview.component.ts [hunk 1]
```diff
@@ -347,7 +347,7 @@ export class DashboardOverviewComponent implements AfterViewInit
                         {
                             this.TrafficLights[month]++;
                         }
-                        else if (type == "Electricity Outage")
+                        else if (type == "Electricity outage")
```
Comment (anru123) [inline libs/admin-app/dashboard/src/lib/dashboard-overview/dashboard-overview.component.ts:350]: I think it should stay as Water Outage and Electricity Outage, other wise the line chart does not work correctly

File: libs/admin-app/dashboard/src/lib/dashboard-overview/dashboard-overview.component.ts [hunk 2]
```diff
@@ -335,7 +335,7 @@ export class DashboardOverviewComponent implements AfterViewInit
                         {
                             this.Pothole[month]++;
                         }
-                        else if (type == "Water Outage")
+                        else if (type == "Water outage")
```
Comment (anru123) [inline libs/admin-app/dashboard/src/lib/dashboard-overview/dashboard-overview.component.ts:338]: '

File: libs/admin-app/dashboard/src/lib/dashboard-overview/dashboard-overview.component.ts [hunk 3]
```diff
@@ -189,7 +189,7 @@ export class DashboardOverviewComponent implements AfterViewInit
                         {
                             this.backgroundColor[a] = 'rgba(142, 198, 63, 0.6)';
                         }
-                        if (ticketTypes[a] == "Electricity Outage")
+                        if (ticketTypes[a] == "Electricity outage")
```
Comment (anru123) [inline libs/admin-app/dashboard/src/lib/dashboard-overview/dashboard-overview.component.ts:192]: '

**Review by anru123 — CHANGES_REQUESTED** [review]

File: libs/admin-app/dashboard/src/lib/dashboard-overview/dashboard-overview.component.ts (hunk 1 above)
Comment (LeonardoWessels) [reply libs/admin-app/dashboard/src/lib/dashboard-overview/dashboard-overview.component.ts:350]: It did have the capital O's when I was working it. I changed it to this to make them work. I'll pull and check if the changes are unnecessary now though.

**Pull request merged**